src/Akeneo/Pim/Enrichment/Component/Product/Builder/ProductBuilder.php

```diff
@@ -52,10 +52,14 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
-    public function createProduct($identifier = null, $familyCode = null)
+    public function createProduct($identifier = null, $familyCode = null, $uuid = null)
     {
         $product = new $this->productClass();
 
+        if (null !== $uuid && $product->getUuid() !== $uuid->toString()) {
```

Comment:
```suggestion
        if (null !== $uuid) {
```
I'd always do it to keep the code simpler